hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/read/TestKeyInputStream.java

```diff
@@ -111,6 +114,57 @@ private void validate(KeyInputStream keyInputStream, byte[] inputData,
   }
 
   @Test
+  /**
+   * This test runs the others as a single test, so to avoid creating a new
+   * mini-cluster for each test.
+   */
```

Comment:
Avoid "dangling javadoc comment" warning:

```suggestion
  /**
   * This test runs the others as a single test, so to avoid creating a new
   * mini-cluster for each test.
   */
  @Test
```